UnityProject/Assets/Scripts/Player/PlayerSync.Client.cs

```diff
@@ -21,7 +21,7 @@ public Vector3IntEvent OnClientTileReached()
 	/// Client predicted state
 	private PlayerState predictedState;
 
-	private Queue<PlayerAction> pendingActions;
+	public Queue<PlayerAction> pendingActions;
```

Comment:
Please keep this private. It should work fine to keep it private even though the code is in a different .cs file because it's still in the same partial class.